Dockerfile

```diff
@@ -14,8 +14,11 @@ ENV ELSTER_DATENLIEFERANT=$elster_datenlieferant
 ENV ELSTER_HERSTELLER_ID=$elster_hersteller_id
 
 WORKDIR /app
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y pcsc-tools pcscd procps unzip curl && \
+RUN echo "APT::Default-Release \"bullseye\";" >> /etc/apt/apt.conf.d/default-release && \
+    echo "deb http://ftp.debian.org/debian bookworm main" >> /etc/apt/sources.list.d/bookworm.list && \
```

Comment:
This adds the bookworm repository to allow us using it in line 20 to install the pcsc-tools etc?

Reply:
This sets bullseye as the default repository so that running `apt-get install `without the -t options continues using bullseye as the default source. You could verify this by building the image (easiest would be to build the first stage only) with and without this change and comparing versions of packages procps, unzip and curl (installed in line 21) inside the resulting container. [In](https://packages.debian.org/bullseye/curl) [all](https://packages.debian.org/bullseye/unzip) [cases](https://packages.debian.org/bullseye/procps) the bullseye version should have been installed.

Reply:
Ah, thanks. The naming of bullseye and bookworm is too close and I mixed them up. That makes sense. Tbh I have not interacted with the apt configuration so far. Is there a reason that we are not running into the problem described [here ](https://unix.stackexchange.com/a/647205) and should use pinning because of that?

Reply:
Good point. I think we're not really affected by this since the debian installation is only going to run in a (relatively) short-living container where no apt upgrade is ever executed or needed, but pinning would still be the better approach and it's easy enough to implement. Commit will come in a bit!